README.md

```diff
@@ -13,21 +13,34 @@ The Open Grid Emissions Dataset can be [downloaded here](https://singularity.ene
 
 To install and run the pipeline on your computer, open anaconda prompt, navigate to the folder where you want to save the repository, and run the following commands:
 
-```
+```bash
 conda install git
 git clone https://github.com/singularity-energy/open-grid-emissions.git
 conda update conda
 cd open-grid-emissions
 conda env create -f environment.yml
 conda activate open_grid_emissions
+```
+A more detailed walkthough of these steps can be found below in the "Development Setup" section.
```

Comment:
In keeping with the instructions for how to download/setup conda below, can you add instructions for how to set up pipenv? Generally since this is an OS project, I like to include basic instructions in case someone is not familiar with using pipenv. Where would you download it? Are there any other steps you need to take before you run `pipenv sync`? etc.

Reply:
It makes sense

Reply:
Done